src/Education/XamlGenerator/XamlOutputElementInfo.cs

```diff
@@ -20,40 +20,37 @@
 
 namespace SonarLint.VisualStudio.Education.XamlGenerator
 {
-    internal partial class RuleHelpXamlBuilder
+    /// <summary>
+    /// Information about a single XAML output element written to the new stream of XAML.
+    /// </summary>
+    internal struct XamlOutputElementInfo
```

Comment:
Is this type now used outside the `RuleHelpXamlBuilder`?

(it might have been in an earlier version of the PR, but it doesn't appear to be now).